java/vector/src/test/java/org/apache/arrow/vector/file/TestArrowFile.java

```diff
@@ -564,6 +566,63 @@ public void testWriteReadNestedDictionary() throws IOException {
   }
 
   @Test
+  public void testWriteReadFixedSizeBinary() throws IOException {
+    File file = new File("target/mytest_fixed_size_binary.arrow");
+    ByteArrayOutputStream stream = new ByteArrayOutputStream();
+    final int numValues = 10;
+    final int typeWidth = 11;
+    byte[][] byteValues = new byte[numValues][typeWidth];
+    for (int i=0; i<numValues; i++) {
```

Comment:
Can you fix the style of this?

Reply:
Sure, I assume you mean the spacing if the for loop, right?

Reply:
Yes